gems/sorbet/lib/gem_loader.rb

```diff
@@ -371,6 +371,15 @@ class Sorbet::Private::GemLoader
       my_require 'sequel'
       my_require 'sequel/sql'
     end,
+    'sequel_pg' => proc do
+      # sequel_pg assumes that it was required by the adapter class in sequel
+      # (i.e., it's not mean to be required manually). But also, sequel lazily
+      # loads the adapter class depending on the scheme of the database being
+      # connected to. Since 'srb init' never only requires and never connects,
+      # we need to manually load the adapter class ourself, which will then
+      # transitively load sequel_pg
+      my_require 'sequel/adapters/postgres'
+    end,
```

Comment:
tl;dr `sequel_pg` is a weird C-extension that's supposed to be mentioned in a Gemfile but isn't required until the user actually connects to a database.